samples/va_intel/va_intel_interop.cpp

```diff
@@ -329,24 +349,31 @@ static float run(const char* fn1, const char* fn2, bool doInterop)
     va_status = vaSyncSurface(va::display, surface_id);
     CHECK_VASTATUS(va_status, "vaSyncSurface");
 
-    dumpSurface(va::display, surface_id, fn1, doInterop);
-
     cv::Size size(CLIP_WIDTH,CLIP_HEIGHT);
+    cv::Size size2(IMAGE_WIDTH,IMAGE_HEIGHT);
     cv::UMat u;
+    cv::UMat u2;
 
     cv::va_intel::convertFromVASurface(va::display, surface_id, size, u);
-    cv::blur(u, u, cv::Size(7, 7), cv::Point(-3, -3));
-    cv::va_intel::convertToVASurface(va::display, u, surface_id, size);
+    cv::va_intel::convertFromVASurface(va::display, surface_id2, size2, u2);
+    cv::resize(u, u2, size2);
+    cv::va_intel::convertToVASurface(va::display, u2, surface_id2, size2);
+
+    dumpSurface(va::display, surface_id2, fn1, doInterop);
+    cv::va_intel::convertFromVASurface(va::display, surface_id2, size2, u2);
+    cv::blur(u2, u2, cv::Size(7, 7), cv::Point(-3, -3));
+    cv::va_intel::convertToVASurface(va::display, u2, surface_id2, size2);
     t.start();
-    cv::va_intel::convertFromVASurface(va::display, surface_id, size, u);
-    cv::blur(u, u, cv::Size(7, 7), cv::Point(-3, -3));
-    cv::va_intel::convertToVASurface(va::display, u, surface_id, size);
+    cv::va_intel::convertFromVASurface(va::display, surface_id2, size2, u2);
+    cv::blur(u2, u2, cv::Size(7, 7), cv::Point(-3, -3));
+    cv::va_intel::convertToVASurface(va::display, u2, surface_id2, size2);
     t.stop();
+    dumpSurface(va::display, surface_id2, fn2, doInterop);
```

Comment:
no. but I dont know why CL and X have to be installed to use cvtColor and imwrite functions.